src/services/Serialization.jl

```diff
@@ -232,8 +232,15 @@ function unpackVariable(
         Dict{Symbol, MeanMaxPPE}()
     end
 
-    smallData = JSON2.read(packedProps["smallData"], Dict{Symbol, SmallDataTypes})
-
+    # Smalldata refactor to metadata
+    if (haskey(packedProps, "metadata"))
```

Comment:
Just to help, on master branch i added additional option in if packedProps is no longer a JSON string inside the top level JSONable object:
https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/blob/685b5f99a4a1c7a933bcf8dd9af81ecb759c0a7a/src/services/Serialization.jl#L348-L360